openapi/v1/filesCurator.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.1.0
+info:
+  description: |-
+    This swagger page describes the filesCurator APIs.
+  title: ODM API
+  version: default-released
+tags:
+  - name: Files as Curator
+paths:
+  /api/v1/as-curator/files/{id}/download:
+    head:
+      description: |+
+        Check if a file exists by its accession.
+
+        This endpoint can be used to check if a file exists and accessible before downloading it.
+      operationId: headFileAsCurator
+      parameters:
+        - description: Unique identifier (accession) of the object.
+          in: path
+          name: id
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
```

Comment:
I don't know the best practices for head request, but maybe it should be 204?

Reply:
<img width="638" alt="image" src="https://github.com/user-attachments/assets/d5ff7bc0-8511-44b2-9e5e-8673f24d8724">

Let's just copycat S3